app-admin/setools/setools-3.3.8-r7.ebuild

```diff
@@ -87,6 +87,7 @@ src_configure() {
 		--with-java-prefix=${JAVA_HOME} \
 		--disable-selinux-check \
 		--disable-bwidget-check \
+		--with-sepol-devel=${ROOT}/usr \
```

Comment:
Is this only used to find the static libsepol.a? just want to be sure runtime is ok.